src/plugins/intel_cpu/src/nodes/executors/acl/acl_fullyconnected.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "acl_fullyconnected.hpp"
+#include "acl_utils.hpp"
+#include "nodes/executors/executor.hpp"
+#include "nodes/executors/memory_arguments.hpp"
+#include "utils/debug_capabilities.h"
+
+namespace ov {
+namespace intel_cpu {
+
+ACLFullyConnectedExecutor::ACLFullyConnectedExecutor(const FCAttrs &attrs, const PostOps &postOps,
+                                                     const MemoryArgs &memory,
+                                                     const ExecutorContext::CPtr context) {
+    aclTensorAttrs.hasLayoutTypeNHWC = memory.at(ARG_SRC)->getDescPtr()->hasLayoutType(LayoutType::nspc);
+    fullyConnectedLayerInfo.weights_trained_layout = getAclDataLayoutByMemoryDesc(memory.at(ARG_WEI)->getDescPtr());
+    fullyConnectedLayerInfo.transpose_weights = !attrs.weightsNonTransposed;
+    if (memory.at(ARG_SRC)->getPrecision() == ov::element::f16) {
+        fullyConnectedLayerInfo.fp_mixed_precision = true;
```

Comment:
@EgorDuplensky ACL write this
```
Use wider accumulators (32 bit instead of 16 for FP16) to improve accuracy.
```
I think it helps for good accuracy on FP16 precision

Reply:
discussed, better to delete